policy-library/policies/templates/legacy/gcp_gke_cluster_location_v1.yaml

```diff
@@ -0,0 +1,79 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-gke-cluster-location-v1
+spec:
+  crd:
+    spec:
+      names:
+        kind: GKEClusterLocationConstraintV1
+      validation:
+        openAPIV3Schema:
+          properties:
+            allowed_locations:
+              description: "Whitelist of regions and/or zones in which GKE clusters
+                can be created, e.g. us-central1-a, us-east1. Multiple locations can
+                be specified. Regions and zones need to be specified separately, as
+                this allows for restricting clusters to be zonal or regional, e.g.
+                whitelisting us-central1 will allow regional clusters to be created,
+                but will not implicitly allow zonal clusters in any us-central1 zone."
+              type: array
+              items:
+                type: string
+  targets:
+    validation.gcp.forsetisecurity.org:
+      rego: |
+        #
+        # Copyright 2019 Google LLC
+        #
+        # Licensed under the Apache License, Version 2.0 (the "License");
+        # you may not use this file except in compliance with the License.
+        # You may obtain a copy of the License at
+        #
+        #     https://www.apache.org/licenses/LICENSE-2.0
+        #
+        # Unless required by applicable law or agreed to in writing, software
+        # distributed under the License is distributed on an "AS IS" BASIS,
+        # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        # See the License for the specific language governing permissions and
+        # limitations under the License.
+        #
+
+        package templates.gcp.GKEClusterLocationConstraintV1
+
+        import data.validator.gcp.lib as lib
+
+        deny[{
+        	"msg": message,
+        	"details": metadata,
+        }] {
+        	constraint := input.constraint
+        	lib.get_constraint_params(constraint, params)
+        	asset := input.asset
+        	asset.asset_type == "container.googleapis.com/Cluster"
+
+        	cluster_location := asset.resource.data.location
+        	target_location := params.allowed_locations
+        	count({cluster_location} & cast_set(target_location)) == 0
+
+        	message := sprintf("Cluster %v is not allowed in the specified location", [asset.name])
+        	metadata := {"resource": asset.name}
+        }
+
+        ###########################
+        # Rule Utilities
```

Comment:
Is this cut off?

Reply:
it is exactly as it is in the original repo: https://github.com/forseti-security/policy-library/blob/master/policies/templates/legacy/gcp_gke_cluster_location_v1.yaml